data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,101 @@ mission "Coalition: Submarines 4"
 			`	When they're done, they hand you <payment>, and Apri'ie follows you to your ship. "Thankful for you putting up with all the last-minute detours I am, Captain. Safe travels, may you have." He does a short gesture as a form of goodbye: you put your forearm on the ground, and he places one of his legs over it. Once you're done, he heads back to the research center.`
+
+mission "Coalition: Interesting Connections"
+	landing
+	name "Interesting Connection"
+	description "A team of Heliarch scientists want to know more about the connection between humans and Hai. Bring them some historic data from Hai Home."
+	to offer
+		has "coalition knows hai"
+	source
+		planet "Ring of Wisdom"
+	stopover Hai-home
+	destination
+		planet "Ring of Wisdom"
+	on offer
+		conversation
+			`After you entered the Spaceport a Kimek is suddenly approaching you. "You, human you are? ... Idiotic question that is, it's obvious. Scientific help i need from you. Of course i will pay you. Will you accept? `
+			choice
+				`	"I need to know what i have to do before i can accept this job."`
+				`	"I'm not looking for work right now"`
+					decline
+			`	"Oh, of course. I will tell you. From an team of Heliarch scientists i am. From a connection between humans and another species we heard. Because this connection another break in the quargs limitations is, we would like to know more about it. Getting some historic data is all you have to do."`
+			`   This would mean to travel to Hai territory. As there is no data in human space that is public.`
+			choice
+				`	"I will do everything for science."`
+				`	"Sorry but that's a too long way for me."`
+					decline
+			`	"Ah thanks to you." He wishes you best luck and goes away.`
+				accept
+	on stopover Hai-home
+		conversation
+			`When you landed on Hai Home you ask some people around for an historic library where you could find the info the Kimek is searching for. To your pleasure it is not taking long for gathering some data and you are ready to get back again.`
+	on complete
+		conversation
+			`After your landing you directly go to the Kimek and give hime the data device. Happy that he got what he was searching for he says "Thanks to you human. I really appretiate your help. Carefully we will scan each detail of this info. Maybe we will need some more another time. Hope i can count on your help then." After he said that he is going away.`
+
+		"coalition jobs" += 1
+		payment 100000
+		event "coalition: interest raises" 12
```

Comment:
It's not necessary to do what I'll explain, but just in case you didn't know and want to make use of it, you can set intervals for these timer events.

For instance, if you wanted the timer to be somewhere between 12 and 20 days, you'd just add a 20 after the 12 (so the line would be `event "coalition: interest raises" 12 20`.

Don't need to do that if you don't want to, just pointing out you can if you didn't already know you could.